content/docs/guides/crosswalk/kubernetes/_index.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Pulumi Crosswalk for Kubernetes"
+linktitle: Crosswalk for Kubernetes
+menu:
+  userguides:
+    parent: guides
+    identifier: crosswalk-kubernetes
+
+aliases: ["/docs/reference/crosswalk/kubernetes/"]
+---
+
+Pulumi Crosswalk for Kubernetes is a collection of industry standard
```

Comment:
Is it worth adding an aside to the effect of

> This guide is for provisioning and configuring production-grade Kubernetes clusters. If you are just getting started with Pulumi and Kubernetes, [the Get Started guide is a better place to start](https://www.pulumi.com/docs/get-started/kubernetes/).

Reply:
Done

Reply:
Done